### ISSUE TITLE
cmd/cacheserver: Access write stall

### ISSUE THREAD

**Issue by masiulaniec** [upspin/upspin#513]
My text editor process has annoyingly entered a lengthy non-interruptible sleep trying to write the new version of an Access file. I issued the write while the cacheserver is busy uploading many large `*_wbf` files. I believe my small write is blocked by these bulky blocks because cacheserver deliberately excludes Access files from the deferred upload optimization (Access files are uploaded eagerly for security reasons). Perhaps cacheserver could include a simple scheduler, for example one that prioritizes upload of tiny blocks?
